src/guc.c

```diff
@@ -453,14 +453,7 @@ _guc_init(void)
 							"Maximum open chunks per insert",
 							"Maximum number of open chunk tables per insert",
 							&ts_guc_max_open_chunks_per_insert,
-							Min(work_mem * INT64CONST(1024) / INT64CONST(25000),
-								PG_INT16_MAX), /* Measurements via
-												* `MemoryContextStats(TopMemoryContext)`
-												* show chunk insert
-												* state memory context
-												* takes up ~25K bytes
-												* (work_mem is in
-												* kbytes) */
+							1024,
```

Comment:
These old measurements are not really relevant, because chunk lookup was redone several times, and we also have multi-insert now which completely changes the memory usage.

Reply:
I think the old default was not neccessarily the best - as I guess `work_mem` should be available to be used for other things - but the formula planned to use all of it to be used for this cache only
note: I wonder if the 25k estimate still stands...this formula was last changed 3 years ago

with the new constant 1024 and using the 25k estimate from the old comment; the max cache size is: `25K*1024 ~ 25M` ; which could be greater than `work_mem` - I think the default `work_mem=4M` 

I don't know if there is any counter infra which could be added to the underlying cache to track hit/miss counts and size so that trashing/etc can be tracked/identified

Reply:
Not sure the size of our various metadata caches should be governed by `work_mem`. It's basically how much memory postgres uses for sort before it starts spilling to disk. The Postgres defaults like 4MB are outdated and don't really apply if we're working with a 1000-chunk (this means a terabyte at least) hypertable. `timescaledb-tune` should configure more sensible values.

Reply:
For context, on my machine `timescaledb-tune` sets `work_mem` to 5 MB :confused: 
The old default value for `timescaledb.max_open_chunks_per_insert` with this setting is 208.

Reply:
> The old default value for timescaledb.max_open_chunks_per_insert with this setting is 208.

Not great, not terrible, still leads to cache thrashing combined with the other old default.